lib/trento/domain/host/commands/deregister_host.ex

```diff
@@ -8,5 +8,6 @@ defmodule Trento.Domain.Commands.DeregisterHost do
 
   defcommand do
     field :host_id, Ecto.UUID
+    field :deregistered_at, :utc_datetime_usec, default: DateTime.utc_now()
```

Comment:
I don't think we need this new field. Actually it is no even used in the process manager code.
We can get the datetime during the execute or apply functions of the aggregate.
Or is it go make it easier to test?

Reply:
I like this solution because this makes the process manager and aggregate logic completely pure, since asking for a DateTime is a side effect. doesn't change much if we store this date instead of the actual deregistration event date since they would differ only by a few milliseconds and we don't need that solution. Doing this, everything is pure and testing is easier.